dev/com.ibm.ws.security.openidconnect.server/resources/com/ibm/ws/security/openidconnect/server/internal/resources/OidcServerMessages.nlsprops

```diff
@@ -223,10 +223,18 @@ BACKCHANNEL_LOGOUT_REQUEST_EXCEPTION.useraction=Ensure that the back-channel log
 # NOTE: Start of new NLS prefix block: CWWKS1950 - CWWKS1999
 #
 
-BACKCHANNEL_LOGOUT_REQUESTS_INVOKE_ALL_FAILED=CWWKS1950E: The {0} OpenID Connect provider encountered an error while submitting back-channel logout requests: {1}
-BACKCHANNEL_LOGOUT_REQUESTS_INVOKE_ALL_FAILED.explanation=Sending requests to the back-channel logout URIs was interrupted, or another error occurred while scheduling the requests for execution.
+BACKCHANNEL_LOGOUT_REQUESTS_INVOKE_ALL_FAILED=CWWKS1950E: The {0} OpenID Connect provider encountered an error on submitting back-channel logout requests: {1}
+BACKCHANNEL_LOGOUT_REQUESTS_INVOKE_ALL_FAILED.explanation=Requests sent to the back-channel logout URIs were interrupted, or another error occurred in scheduling the requests for execution.
```

Comment:
Reviewed explanation:
Requests sent to the back-channel logout URIs were interrupted, or another error occurred in scheduling the requests to run.